aiida/cmdline/params/types/identifier.py

```diff
@@ -59,7 +59,10 @@ def __init__(self, sub_classes=None):
                 try:
                     entry_point = get_entry_point_from_string(entry_point_string)
                 except (ValueError, exceptions.EntryPointError) as exception:
-                    raise ValueError('{} is not a valid entry point string: {}'.format(entry_point_string, exception))
+                    raise ValueError(
+                        '{} is not a valid entry point string: {}\n'.format(entry_point_string, exception) +
+                        'Try running `reentry scan` to update the entry point cache.'
```

Comment:
This is not the only place where this can happen. We should probably move this warning into the base loading function `aiida.plugins.load_plugin` or something. What do you think?

Reply:
Thanks, yes I just put this fix in quickly during the hackathon in order not to forget.
This should be handled on a higher level - done now.